buildAndRunSite.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-jekyll serve --incremental --watch --config _config.yml,_config_en.yml 
\ No newline at end of file
+jekyll serve --incremental --watch --config _config_default.yml,_config_en.yml
```

Comment:
Just ran it and it does not generate `es`, but makes `en` default - which the previous option did